challenge-8/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+    <script>
+      const yesterday = (a, b) => {
```

Comment:
Is the function name giving us `yesterday`'s date? I think this needs to be named better, but otherwise it looks fine.

Reply:
Yeah, uh... basically the Beatles song "Yesterday" was originally titled "Scrambled Eggs", so it was kind of an inside joke with myself. But I get that if I was working with another person it would have to be something else.